_posts/2021-02-11-asking-for-help.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: epic
+title: "Why Asking for Help Strengthens Engineering Teams"
+date: 2021-02-11
+categories: [team, culture, community, learning, teaching]
+author: anna
+canonical: https://medium.com/swlh/why-asking-for-help-strengthens-engineering-teams-6b1eadc86936
+---
+
+I know that for many developers, especially those early in their careers, asking for help can be intimidating. I
+often fear wasting someone’s time or exposing myself as less skilled or smart than my team initially thought.
+
+In my first month as a software engineer at Artsy (and barely six months into life as an engineer after
+[transitioning from a career in communications](https://medium.com/swlh/how-losing-my-job-helped-me-find-my-way-back-8c8f86552acc)),
+I was struggling through a ticket assigned to me as a “good first issue.” (The team estimated the task to be
+straightforward enough for someone new to the team.) After a few hours stumbling between the ticket, my code, and
+Google, I made very little progress.
+
+<!-- more -->
+
+Early and often, my fellow engineers at Artsy encourage me to ask for help when I hit roadblocks. Even with the
+support explicitly offered to me, I was shaky. The extra 10 seconds it took to type out my question in Slack was
+just enough time for me to chicken out.
+
+Opening up with my team members about my concerns, I learned that asking for help is not just for me but ultimately
+benefits my team. Sharing their insights below, I hope other engineers, early-career or not, feel empowered to ask
+questions, stay curious, and prioritize learning for themselves and their team — all hallmarks of great engineers.
+
+##Why It’s So Hard to Ask For Help
+
+Working in tech and as an engineer (and even in
+[undergraduate computer science classes](https://medium.com/better-programming/why-women-dont-pursue-software-engineering-17d0dead24e9),)
```

Comment:
**minor suggestion**: It might help improve readability within the markdown file to extract these URLs from Markdown inline to [reference links](https://daringfireball.net/projects/markdown/syntax#link)!

For example:

> Working in tech and as an engineer (and even in [undergraduate computer science classes][1]), I often felt surrounded by people smarter than me.

[1]: https://medium.com/better-programming/why-women-dont-pursue-software-engineering-17d0dead24e9

```markdown
Working in tech and as an engineer (and even in [undergraduate computer science classes][1]), I often felt surrounded by people smarter than me.

[1]: https://medium.com/better-programming/why-women-dont-pursue-software-engineering-17d0dead24e9
```

You can also give these reference links more meaningful names:

```markdown
```markdown
Working in tech and as an engineer (and even in [undergraduate computer science classes][why-women-dont-pursue-software-engineering]), I often felt surrounded by people smarter than me.

[why-women-dont-pursue-software-engineering]: https://medium.com/better-programming/why-women-dont-pursue-software-engineering-17d0dead24e9
```
```

Reply:
I like this! I'll give it a shot and let you know if I have questions.

Reply:
@dblandin My current changes produce the below. Was following [this](https://www.markdownguide.org/basic-syntax/#links). Are my eyes missing something?? I assume they're not supposed to show up at all.

<img width="803" alt="Screen Shot 2021-02-11 at 8 16 12 AM" src="https://user-images.githubusercontent.com/9466631/107656837-c93fae80-6c41-11eb-8934-884f9222c1b0.png">

Reply:
Looks like there's just a bit of whitespace breaking the reference links.

When _defining_ reference links:

- Ensure that the reference is on a single line
- The reference value should be the raw URL (no wrapping in angle brackets <>)

```markdown
[how-losing-my-job-helped-me-find-my-way-back]: https://medium.com/swlh/how-losing-my-job-helped-me-find-my-way-back-8c8f86552acc
```

When _using_ a reference link:

- Ensure that the link does not space multiple lines
- Use the following [display][reference] syntax
- Ensure that there's no whitespace between the two bracketed `[][]` parts

```markdown
[Harvard Business Review][people-are-willing-to-earn-less-money-to-do-more-meaningful-work]
```

That should fix the formatting!

Reply:
YES it did thank you!